source/Threema/MsgApi/PublicKeyStore.php

```diff
@@ -62,4 +63,11 @@
 	 * @return bool
 	 */
 	abstract protected function savePublicKey($threemaId, $publicKey);
+    
+	/**
+	 * Initialize a new File|PhpFile Public Key Store
+	 * @param string $path
+	 * @return File|PhpFile
+	 */
+	abstract protected function create($path);
```

Comment:
Okay, I think it is bad practice to add abstract static classes. 
So I think just removing the abstract function is a good way to go. Could you please do that? (and do not forget to fix the "whitespace issue" reported by Codacy)